src/precice/SolverInterface.hpp

```diff
@@ -808,6 +808,27 @@ class SolverInterface {
    * on the receiving side, since the associated data values of the
    * filtered vertices are filled with zero data.
    *
+   * @note This function can only be called once per participant and
+   * rank and an assert is thrown, if you try to call it more than once.
```

Comment:
API usage should __never__ result in an assert.

Reply:
Fixed the wording.